napari/layers/surface/surface.py

```diff
@@ -592,12 +592,12 @@ def _has_texture(self) -> bool:
             and len(self.texcoords)
         )
 
-    def _get_state(self) -> dict:
+    def _get_state(self) -> dict[str, Any]:
         """Get dictionary of layer state.
 
         Returns
         -------
-        state : dict
+        state : dict[str, Any]
```

Comment:
Looking at numpydoc reference, it should look like 
```suggestion
        state : dict of str to Any
```

https://numpydoc.readthedocs.io/en/latest/format.html#parameters

If you agree, please update all type annotations

Reply:
Seems fine. Will do.